backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -2096,12 +2097,25 @@ class ContentModifyAbstractSchema(marshmallow.Schema):
     )
 
 
+class ContentModifyNamespaceAbstractSchema(marshmallow.Schema):
+    content_namespace = StrippedString(
+        required=False,
+        description="Content_namespace of the object, raw text",
+    )
```

Comment:
Relative to my other comment about my opinion about required value instead of optionnal.

Reply:
done